akka-remote/src/main/scala/akka/remote/Remoting.scala

```diff
@@ -549,6 +549,20 @@ private[remote] class EndpointManager(conf: Config, log: LoggingAdapter) extends
         case (Some((endpoint, currentUid)), Some(quarantineUid)) if currentUid == quarantineUid ⇒ context.stop(endpoint)
         case _ ⇒ // nothing to stop
       }
+
+      // Stop all stashed pending connections
+      pendingReadHandoffs = pendingReadHandoffs.filterNot {
+        case (pendingActor, pendingHandle) ⇒
+          val addressMatches = pendingHandle.remoteAddress == address
+          // This yields true also if the Quarantine command did not contain a UID
+          val uidMatches = uidToQuarantineOption.forall(_ == pendingHandle.handshakeInfo.uid)
+
+          // Side-effecting here
+          context.stop(pendingActor)
```

Comment:
is it to correct to stop all pending actors, independent of the quarantine address and uid?